CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Ensuring that the random number generator is TRNG. (#1200)
 - ESP32-C6: Add timer wakeup source for deepsleep (#1201)
 - Introduce `InterruptExecutor::spawner()` (#1211)
+- Creating multichip docs with GitHub Pages (#1243)
```

Comment:
Documentation does not affect the user-facing API in any way, there is no need for a `CHANGELOG.md` entry in this PR.